tests/test_image_verifier.py

```diff
@@ -21,179 +21,314 @@
+                            "tags": [{"name": "1-6.1"}],
+                            "edges": {"repository": {"data": repos[0]}},
+                        }
+                    ],
+                }
+            )
         ]
         ret = self.verifier.verify_image("foo-1-1")
         self.assertEqual(ret, {"foo-1-1": ["content-set"]})
 
     def test_get_verify_image_no_repo(self):
-        self.lb.find_container_repositories.return_value = []
+        self.pgql.find_images_by_nvr.return_value = [
+            ContainerImage(
+                {"brew": {"build": "foo-1-1"}, "content_sets": ["content-set"]}
+            )
+        ]
         self.assertRaisesRegex(
-            ValueError, r'Cannot get repository.*',
-            self.verifier.verify_image, "foo/bar")
+            ValueError,
+            r"Cannot get repository for image.*",
+            self.verifier.verify_image,
+            "foo-1-1",
+        )
 
     def test_get_verify_image_multiple_repos(self):
-        self.lb.find_container_repositories.return_value = ["foo", "bar"]
+        self.pgql.find_repositories_by_repository_name.return_value = [
+            ContainerRepository(
+                {
+                    "release_categories": ["Generally Available"],
+                    "auto_rebuild_tags": ["latest"],
+                    "registry": "registry.access.redhat.com",
+                    "repository": "foo",
+                }
+            ),
+            ContainerRepository(
+                {
+                    "release_categories": ["Generally Available"],
+                    "auto_rebuild_tags": ["latest"],
+                    "registry": "registry.access.redhat.com",
+                    "repository": "bar",
+                }
+            )
+        ]
+        self.pgql.find_images_by_nvr.return_value = [
+            ContainerImage(
+                {
+                    "brew": {"build": "foo-1-1"},
+                    "content_sets": ["content-set"],
+                    "repositories": [
+                        {
+                            "registry": "registry.access.redhat.com",
+                            "repository": "foo",
+                            "published": True,
+                            "tags": [{"name": "1-6.1"}],
+                        },
+                        {
+                            "registry": "registry.access.redhat.com",
+                            "repository": "bar",
+                            "published": True,
+                            "tags": [{"name": "1-6.1"}],
+                        },
+                    ],
+                }
+            )
+        ]
         self.assertRaisesRegex(
-            ValueError, r'.*found in multiple repositories in Lightblue.',
-            self.verifier.verify_image, "foo/bar")
+            ValueError,
+            r".*found in multiple repositories in Pyxis.",
+            self.verifier.verify_image,
+            "foo-1-1",
```

Comment:
@qixiang Previously there was a "foo/bar" argument, but I think the argument "foo-1-1", which is the image's brew build, makes more sense

Reply:
right